crates/bevy_math/src/ops.rs

```diff
@@ -23,6 +20,10 @@
 // - `f32::ln_gamma`
 
 #[cfg(not(feature = "libm"))]
+#[expect(
+    clippy::disallowed_methods,
+    reason = "Many of the disallowed methods point to methods which are ultimately defined here."
```

Comment:
I'd suggest an (imo clearer) version.
```suggestion
    reason = "Many of the disallowed methods are disallowed to force code to use the feature-conditional re-exports from this module, but this module itself is exempt from that rule."
```